File: python/ZLICZANIE.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

#  


def main(args):
    a = int(input("Podaj liczbę:"))
    licz5 = 0
    licz7 = 0
    while a !=0:
        if a % 5 == 0:
            licz5= licz5 + 1
        if a % 7 == 0:
            licz7= licz7 + 1
        a = int(input("Podaj liczbę:"))
            
    print("Podzielnych przez 5:, licz5")
    print("Podzielnych przez 7:, licz7")
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: C++/zliczanie.cpp
/*
 * zliczanie.cpp
 *
 */


#include <iostream>
using namespace std;


int main(int argc, char **argv)
{
	int a;
    int licz5 = 0 ;
    int licz7 = 0 ;
    
    cout << "Podaj liczbę:";
    cin >> a;
    while  (a != 0) {
        if (a % 5 == 0) {
            licz5= licz5 + 1;
        }
        if (a % 7 == 0) {
            licz7= licz7 + 1;
        }
        cout << "Podaj liczbę:";
        cin >> a;
    }   
     cout<< "Podzielnych przez 5: " << licz5 << endl;
     cout<< "Podzielnych przez 7: " << licz7 << endl;
	return 0;
}

